### ISSUE TITLE
Project: Produce Outreach Material

### ISSUE THREAD

**Issue by jamesmcgirk** [smartcontractresearchforum/research_team#112]
### Project Description

We need documents that will allow us to quickly communicate SCRF's mission and how to get involved with the organization. Versions of the one-pager and press kit should be update with new projects we've developed and the language that we've developed from the branding exercise. 

*Producing (or rather, adapting) a one-page document and a longer press kit with information about the Smart Contract Research Forum (SCRF) for outreach purposes.  

### Work Plan

*Enter the steps as reflected in your Work Plan:*

- [x] Gather existing materials
- [x] Complete Branding Exercise
- [ ] Draft one-pager 
- [ ] Draft packet
- [ ] Present to Outreach Team
- [ ] Corrections, Additions
- [ ] Test Drive
- [ ] Mark Down

### Resources

- [link to project plan](#linkme)
- [link to process doc/proof of concept](https://docs.google.com/document/d/1yGMG7UIPHcnUggKtb9T6nouSCYZO5Lwu5MmqgGMXyNU/edit#
- [link to any other relevant supporting materials](#linkme)

### General Tips

- [SCRF Research Improvement Proposal](https://github.com/smartcontractresearchforum/research_team/blob/main/docs/SCRF_Research_Improvement_Proposal.md)
- [SCRF Research Improvement Process](https://github.com/smartcontractresearchforum/research_team/blob/main/docs/SCRF_Research_Improvement_Process.md)

_**Provide updates to this ticket at regular intervals**. These tickets need to represent an accurate audit trail while contributing to the institutional record of SCRF._

_Ensure the correct tags and projects have been selected in the sidebar -->_

**Comment by jamesmcgirk** [comment]
Once the branding exercise is completed, I'll refresh/rewrite the one-pager and then produce a more extensive pack with information about grants programs.

**Comment by eleventh19** [comment]
currently blocked by the Branding Exercise #113

**Comment by jamesmcgirk** [comment]
@jamesmcgirk add in the draft document that Eugene created

**Comment by jamesmcgirk** [comment]
Final version due 7/30 (Summit permitting). Adding in [Temporary Branding Exercise Document](https://docs.google.com/document/d/1yGMG7UIPHcnUggKtb9T6nouSCYZO5Lwu5MmqgGMXyNU/edit#) as POC

**Comment by eleventh19** [comment]
this is part of the #155 project, I need to tag Maria here

**Comment by jamesmcgirk** [comment]
I'll close since this has been merged into 155
